npm/test-browser.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+// ---------------------------------------------------------------------------------------------------------------------
+// This script is intended to execute all unit tests in the Chrome Browser.
+// ---------------------------------------------------------------------------------------------------------------------
+
+require('shelljs/global');
+require('colors');
+
+var path = require('path'),
+    Server = require('karma').Server,
+
+    KARMA_CONFIG_PATH = path.join(__dirname, '..', 'karma.conf');
+
+module.exports = function (exit) {
+    console.log('Running unit tests within browser...'.yellow);
+    new Server({
```

Comment:
Except JSDoc of course, which cannot be used as a library [presently](https://github.com/jsdoc3/jsdoc/issues/938).